recipes/unumpy/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "unumpy" %}
+{% set version = "0.5.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Quansight-Labs/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 07777c96c0d559b27739f64f91c286842fc99ae409df61bd0d9f3c49c2a4c4f6
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+  run:
+    - python
```

You also need this in run
```suggestion
    - python >=3.5
```